src/lug/Main/MainAndroid.hpp

```diff
@@ -2,15 +2,19 @@
 
 #include <android/configuration.h>
 #include <android/native_activity.h>
+#include <lug/Window/Window.hpp>
 
 extern int main(int argc, const char* argv[]);
 
 namespace lug {
 namespace Main {
 
-class LugAndroidApp {
+extern std::queue<lug::Window::Event> events;
```

Comment:
I spent 1 hour trying to compile on android with the extern in lug/Window/Android/WindowImplAndroid.cpp with no success , ended up putting in here and it works